src/controllers/controller.polarArea.js

```diff
@@ -271,7 +267,46 @@ module.exports = DatasetController.extend({
 
 		return resolve([
 			me.chart.options.elements.arc.angle,
-			(2 * Math.PI) / count
+			angle
 		], context, index);
+	},
+
+	/**
+	 * if no angles options, circumference = 2*PI and each arc will be (2*PI)/arcCount
+	 * if angles options, circumference = sum(angles)
+	 *    if angles.length < arcCount => undefined angles will be set to (2*PI)/arcCount,
+	 *                                   defined angles will be recompute to fill circumference-sum(undefinedAngles) stay proportional to original values
+	 *    if angles.length > arcCount => circumference remains the same and existing angles will be recompute to stay proportional to original values
+	 * @private
+	 */
+	_convertAngles: function(angles) {
+		var me = this;
+		var dataset = me.getDataset();
+		var meta = me.getMeta();
+		var angle, ratio;
+		var initialCircumference = 0;
+		var convertedAngles = [];
+		var circumference = angles.length ? angles.reduce(function(a, b) {
```

Comment:
What happens if the sum of the angles is > 2*pi?

Reply:
1) If the sum of the angles is > 2*pi, the draw is more than one complete circle (you can easily try it here : https://jsfiddle.net/wanvfqbr/ ). 
This does not chock me as it is the same behavior if sum of the angles is < 2*pi (draw is less than one complete circle) and people may find this usefull.
Tell me if you would expect another behavior.

2) How much test do you think we need ? I'd suggest four cases : 
* angles option is not set
* options.angles.length = (visible datasets count)
* options.angles.length < (visible datasets count)
* options.angles.length > (visible datasets count)